### PR TITLE
Feature/at 8567 ditco upload signed docs

### PR DESCRIPTION
**Pull request by DGabriel-CCPO**
When the user selects DICTO as their contracting office, it will trigger submission flow to submit the acquistion package to DITCO.

### CODE REVIEW

File: src/components/ATATFileUpload.vue [hunk 1]
```diff
@@ -322,13 +329,21 @@ export default class ATATFileUpload extends Vue {
...
-
+    if(_validFiles.length > this.maxNumberOfFiles){
+      this.moreThanMax = true
+      const errorText = this.maxNumberOfFiles === 1?
+        `Too many files selected. You can upload up to ${this.maxNumberOfFiles} file.`
+        :`Too many files selected. You can upload up to ${this.maxNumberOfFiles} files.`
+      this.$refs.atatFileUpload.errorBucket
+        .push(errorText)
+      this.setErrorMessage()
+      return
```
Comment (tonysumner-ccpo) [inline src/components/ATATFileUpload.vue:343]: this stops the flow of execution and is returning undefined.
Just doublechecking that this is what you want to do.

File: src/router/resolvers/index.ts [hunk 2]
```diff
@@ -58,15 +58,15 @@ export const CreateEvalPlanRouteResolver = (current: string): string => {
...
-export const UploadJAMRRDocumentsRouteResolver = (current: string): string => {
-  const isDitco = AcquisitionPackage.contractingShop === 'DITCO'
-  if(current === routeNames.ReadyToGeneratePackage && (evalPlanRequired()|| isDitco)){
-    return routeNames.FinancialPOCForm
-  }
-  return !evalPlanRequired()
-    ? routeNames.UploadJAMRRDocuments 
-    : routeNames.ReadyToGeneratePackage;
-};
+// export const UploadJAMRRDocumentsRouteResolver = (current: string): string => {
```
Comment (tonysumner-ccpo) [inline src/router/resolvers/index.ts:61]: remove this....

File: src/router/resolvers/index.ts [hunk 3]
```diff
@@ -1329,7 +1329,7 @@ const routeResolvers: Record<string, StepRouteResolver> = {
   EvalPlanDetailsRouteResolver,
   ArchitecturalDesignDetailsRouteResolver,
   SecurityRequirementsResolver,
-  UploadJAMRRDocumentsRouteResolver,
+  // UploadJAMRRDocumentsRouteResolver,
```
Comment (tonysumner-ccpo) [inline src/router/resolvers/index.ts:1332]: remove this, too.

File: src/router/stepper.ts [hunk 4]
```diff
@@ -163,7 +164,7 @@ import {
   EvalPlanDetailsRouteResolver,
   ArchitecturalDesignDetailsRouteResolver,
   SecurityRequirementsResolver,
-  UploadJAMRRDocumentsRouteResolver,
+  // UploadJAMRRDocumentsRouteResolver,
```
Comment (tonysumner-ccpo) [inline src/router/stepper.ts:167]: remove this

File: src/router/stepper.ts [hunk 5]
```diff
@@ -1093,15 +1095,15 @@ export const stepperRoutes: Array<StepperRouteConfig> = [
...
-      {
-        menuText: "Upload J&A and MRR Documents",
-        path:"upload-ja-mrr-documents",
-        excludeFromMenu: true,
-        name: routeNames.UploadJAMRRDocuments,
-        completePercentageWeight: 0,
-        component: UploadJAMRRDocuments,
-        routeResolver: UploadJAMRRDocumentsRouteResolver,
-      },
+      // {
```
Comment (tonysumner-ccpo) [inline src/router/stepper.ts:1098]: remove this

**Review by ericyoungquist-ccpo — APPROVED** [review]

**Pull request merged**